packages/bisheng/src/utils/source-data.js

```diff
@@ -48,10 +48,12 @@ function getPropPath(filename, sources) {
 
 function filesToTreeStructure(files, sources) {
   const cleanedSources = sources.map(source => source.replace(/^\.?\//, ''));
-  return files.reduce((filesTree, filename) => {
+  const filesTree = files.reduce((filesTree, filename) => {
     const propLens = R.lensPath(getPropPath(filename, cleanedSources));
     return R.set(propLens, filename, filesTree);
   }, {});
+  // fix: windows platform
+  return filesTree[""] || filesTree
```

Comment:
[Here](https://github.com/benjycui/bisheng/blob/master/packages/bisheng/src/utils/source-data.js#L50) and [here](https://github.com/benjycui/bisheng/blob/master/packages/bisheng/src/utils/source-data.js#L46) don't consider Windows' `\path-to\filename.extension`.

So, I think you need to update RegExp, instead of hacking `return filesTree[""] || filesTree`